src/mavlink_tcp.h

```diff
@@ -60,6 +63,6 @@ class Mavlink_TCP : public Pollable {
 	void (*_highres_imu_msg_callback)(const mavlink_highres_imu_t *msg, void *data) = NULL;
 	const void *_highres_imu_msg_callback_data;
 
-	const uint8_t _system_id = 100;
-	const uint8_t _component_id = 1;
+	const uint8_t _system_id = 1; // TODO default is 1, but check with heartbeat
```

Comment:
Why change the system id of aero-optical-flow to 1? Also setting the component id to MAV_COMP_ID_CAMERA do not look right.

Reply:
Because it belongs to the same system as the flight controller but is another componant. The default system id is 1 (also in this case). Componant 1 is the flight controller itself. I went for `MAV_COMP_ID_CAMERA` because there is no companion computer or something similar and basically, optical flow is a camera. See http://mavlink.org/messages/common under `MAV_COMPONENT`